aws_lambda_builders/workflows/python_pip/compat.py

```diff
@@ -1,13 +1,21 @@
 import os
 
+from aws_lambda_builders.workflows.python_pip.utils import OSUtils
 
-def pip_import_string():
-    import pip
-    pip_major_version = pip.__version__.split('.')[0]
+
+def pip_import_string(python_exe):
+    os_utils = OSUtils()
+    cmd = [
+        python_exe,
+        "-c",
+        "import pip; assert pip.__version__.split('.')[0] == 9"
```

Comment:
Shouldn't this be <= 9? Otherwise we try to do `from pip._internal import main` for any pip version that isn't 9, which I don't think the intention is correct.

9 should be a string here as well

Reply:
This is a great catch, thanks! yes will update it.